### ISSUE TITLE
[Cosmetic][Quest][Ammen Vale] An Alternative Alternative script missing

### ISSUE THREAD

**Issue by ashphoenix92** [dalaranwow/dalaran-wow#5517]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Upone completeting the quest no scripted event happens.

**Expected behaviour**:  Cut-scene styled event happens.
On completion of this quest, Daedal gives the remedy to the elven patient.

Daedal says: This should do it...
Daedal says: Wake up little creature. Wake up from your slumber... You are safe now.
Injured Night Elf Priestess groans.
Injured Night Elf Priestess sits up.
Injured Night Elf Priestess says: Whe... Where am I? What... What is this place?
Injured Night Elf Priestess says: BY THE LIGHT OF ELUNE! Eredar! Cursed demons, what have you done to me? I have not come alone, we will fight you as we have in the past! I...
Injured Night Elf Priestess falls back into a coma.
Anchorite Fateema says: Man'ari... But how... We are not eredar! She must know! If there are others like her, we must find them and explain...

1 None. Just quest complete the quest as regular and event does seem to be unscripted

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

http://www.wowhead.com/quest=9473/an-alternative-alternative

https://wow.gamepedia.com/Quest:An_Alternative_Alternative

**Include Screenshots from the issue if necessary**
